strings/soapstone_suffixes.json

```diff
@@ -0,0 +1,231 @@
+		"monkey",
+		"xeno",
+		"slimeperson",
+		"snowflake",
+		"enemy",
+		"tough enemy",
+		"Nanotrasen",
+		"Syndicate",
+		"skeleton",
+		"flier",
+		"statue",
+		"monster",
+		"strange creature",
+		"demon",
+		"boss",
+		"dark spirit",
+		"saint",
+		"wretch",
+		"charmer",
+		"miscreant",
+		"liar",
+		"fatty",
+		"beanpole",
+		"merchant",
+		"prisoner"
+	],
+
+	"Careers": [
+		"assistant",
+		"captain",
+		"head of personnel",
+		"head of security",
+		"warden",
+		"security officer",
+		"detective",
+		"chief engineer",
+		"engineer",
+		"atmos tech",
+		"research director",
+		"scientist",
+		"robotics",
+		"chief medical officer",
+		"doctor",
+		"chemist",
+		"geneticist",
+		"virologist",
+		"janitor",
+		"bartender",
+		"cook",
+		"clown",
+		"mime",
+		"chaplain",
+		"librarian",
+		"quartermaster",
+		"cargo tech",
+		"miner",
+		"lawyer",
+		"AI",
+		"cyborg",
+		"drone",
+		"pAI"
+	],
+	
+	"Antagonists": [
+		"traitor",
+		"changeling",
+		"nuke op",
+		"wizard",
+		"alien",
+		"ghost",
+		"blob",
+		"cultist",
+		"devil"
+	],
+	
+	"Objects": [
+		"airlock",
+		"table",
+		"rack",
+		"weapon",
+		"trap",
+		"false wall",
+		"window",
+		"item",
+		"loot",
+		"teleporter",
+		"blueprints",
+		"boots",
+		"clothes",
+		"clothing",
+		"drugs",
+		"medicine",
+		"crate",
+		"locker",
+		"handcuffs",
+		"amazing item",
+		"amazing loot",
+		"amazing clothing",
+		"amazing drugs",
+		"amazing rack"
+	],
+	
+	"Techniques": [
+		"close-ranged battle",
+		"ranged battle",
+		"eliminating one at a time",
+		"luring it out",
+		"beating to a pulp",
+		"lying in ambush",
+		"stealth",
+		"fleeing",
+		"charging",
+		"stabbing in the back"
+	],
+	
+	"Actions": [
+		"attacking",
+		"hacking",
+		"sneaking",
+		"slipping",
+		"running",
+		"walking",
+		"sprinting",
+		"running away",
+		"holding with both hands",
+		"jumping",
+		"restraining",
+		"crying",
+		"cheering"
+	],
+	
+	"Geography": [
+		"hallway",
+		"room",
+		"corridor",
+		"airlock",
+		"space",
+		"hidden path",
+		"gorgeous view",
+		"open area",
+		"tight spot",
+		"tunnel",
+		"dark area",
+		"bright area",
+		"shuttle",
+		"pod",
+		"security",
+		"arrivals",
+		"escape",
+		"engineering",
+		"atmospherics",
+		"medbay",
+		"science",
+		"brig",
+		"prison",
+		"asteroid",
+		"cargo",
+		"mining",
+		"cave",
+		"maintenance"
+	],
+	
+	"Orientation": [
+		"front",
+		"back",
+		"left",
+		"right",
+		"up",
+		"down",
+		"north",
+		"south",
+		"east",
+		"west",
+		"northeast",
+		"northwest",
+		"southeast",
+		"southwest",
+		"that way"
+	],
+	
+	"Body parts": [
+		"head",
+		"neck",
+		"stomach",
+		"back",
+		"arm",
+		"leg",
+		"heel",
+		"rear",
+		"tail",
+		"wings",
+		"anywhere",
+		"everywhere"
+	],
+	
+	"Concepts": [
+		"chance",
+		"hint",
+		"secret",
+		"happiness",
+		"sorrow",
+		"life",
+		"death",
+		"elation",
+		"grief",
+		"hope",
+		"despair",
+		"light",
+		"dark",
+		"bravery",
+		"resignation",
+		"comfort",
+		"tears"
+	],
+	
+	"Musings": [
```

Comment:
That's intentional. If people want to write things that make no sense, all the power to them.